src/pages/younghyunHan/Login/Login.js

```diff
@@ -1,7 +1,85 @@
 import './Login.scss';
+import { useNavigate } from 'react-router-dom';
+import { useState } from 'react';
 
 function Login() {
-  return <></>;
+  const [state, setState] = useState('');
+  const [passwordValue, setPasswordValue] = useState('');
+  const [disabled, setDisabled] = useState(true);
+  const [style, setStyle] = useState({ opacity: 0.5, cursor: 'default' });
+
+  function handleIdInput(event) {
+    if (event.target.type === 'text') {
+      setState(event.target.value);
+    } else if (event.target.type === 'password') {
+      setPasswordValue(event.target.value);
+    }
+    handleButton();
+  }
+
+  const checkId = idValue => {
+    return idValue.includes('@') ? true : false;
+  };
+
+  const checkPw = pwValue => {
+    return pwValue.length >= 5 ? true : false;
+  };
+
+  const handleButton = () => {
+    const isValidId = checkId(state);
+    const isValidPw = checkPw(passwordValue);
+
+    if (isValidId && isValidPw) {
+      buttonColor(true);
+    } else {
+      buttonColor(false);
+    }
+  };
+
+  const buttonColor = btnValid => {
+    setDisabled(!btnValid);
+    setStyle({
+      opacity: btnValid ? 1 : 0.5,
+      cursor: btnValid ? 'pointer' : 'default',
+    });
+  };
+
+  const navigate = useNavigate();
+
+  const goToMain = () => {
+    navigate('/main-han');
+  };
+
+  return (
+    <div className="login">
+      <form className="loginBox">
+        <h1>Westagram</h1>
+        <input
+          className="userId"
+          type="text"
+          placeholder="전화번호, 사용자 이름 또는 이메일"
+          onChange={handleIdInput}
+        />
+        <input
+          className="userPassword"
+          type="password"
+          placeholder="비밀번호"
+          onChange={handleIdInput}
+        />
+        <button
+          type="submit"
+          className="loginButton"
+          onClick={goToMain}
+          disabled={disabled}
+          style={style}
```

Comment:
```suggestion
          className={`loginBtn ${
              idValue.indexOf('@') > -1 && pwValue.length >= 5
                ? 'loginBtnLive'
                : ''
            }`}
```
 className의 동적 사용을 적용하면 좋을 것 같아요
인라인 스타일의 우선 순위가 높아서, 팀원이 class를 수정했을 때
class가 적용이 안되는 문제가 있다고 해요. 

조건 연산자를 활용해서 className을 동적으로 쓸 수 있음
className={원래 클래스 +활성화 클래스 <-- 조건 연산자로 클래스 추가 여부 결정}

Reply:
김혜진 멘토님 리뷰 반영하기

Reply:
+삼항연산자의 조건은 위 리뷰처럼 변수로 만들어서 사용하기